Cargo.lock

```diff
@@ -1883,31 +1931,33 @@ dependencies = [
 [[package]]
 name = "zcash_primitives"
 version = "0.5.0"
-source = "git+https://github.com/zcash/librustzcash.git?rev=7801dddf35ca247345cf4f5c5e48791297cad531#7801dddf35ca247345cf4f5c5e48791297cad531"
+source = "git+https://github.com/zcash/librustzcash.git?rev=a01290869a4f0c4e05bdcca550795ea0e76e8550#a01290869a4f0c4e05bdcca550795ea0e76e8550"
 dependencies = [
  "aes",
  "bip0039",
  "bitvec",
  "blake2b_simd 1.0.0",
  "blake2s_simd 1.0.0",
  "bls12_381",
+ "bs58",
  "byteorder",
  "chacha20poly1305",
  "equihash",
  "ff",
  "fpe",
  "group",
+ "hdwallet",
  "hex",
  "incrementalmerkletree",
  "jubjub",
  "lazy_static",
- "log",
  "memuse",
  "nonempty",
  "orchard",
- "pasta_curves",
  "rand",
  "rand_core 0.6.3",
+ "ripemd160",
```

Comment:
This should change to "ripemd" when it's updated after comments on zcash/librustzcash#341 have been addressed.